syntax/package_test.go

```diff
@@ -32,10 +32,18 @@ func TestPackageImport(t *testing.T) {
 	AssertCodesEvalToSameValue(t, `{1, 4, 9, 16}`, `//./examples/simple/simple`)
 }
 
+func TestPackageImportWithQuote(t *testing.T) {
+	AssertCodesEvalToSameValue(t, `{1, 4, 9, 16}`, `//./'examples/simple/simple'`)
+}
+
 func TestPackageImportFromRoot(t *testing.T) {
 	AssertCodesEvalToSameValue(t, `{1, 4, 9, 16}`, `///examples/simple/simple`)
 }
 
+func TestJsonPackageImport(t *testing.T) {
+	AssertCodesEvalToSameValue(t, `{'location': 'Melbourne', 'name': 'foo'}`, `///examples/json/foo.json`)
```

Comment:
```suggestion
	AssertCodesEvalToSameValue(t, `{'location': 'Melbourne', 'name': 'foo'}`, `//./examples/json/foo.json`)
```
Tests seem to be failing for me locally when I run make. I think you're missing a dot here?

Reply:
Hi Cam, we have already add support for /// and //./ to be able to import from root, both should work. You can have a look at the line 35 to 41. 
btw, what is your test failing message?

Reply:
https://github.com/arr-ai/arrai/pull/48 we already have the import from module root.